setup.py

```diff
@@ -20,8 +20,9 @@
     author_email="stac@radiant.earth",
     url="https://github.com/stac-utils/pystac",
     packages=find_packages(),
+    package_data={p: ["py.typed"] for p in find_packages()},
     py_modules=[splitext(basename(path))[0] for path in glob("pystac/*.py")],
-    include_package_data=False,
+    include_package_data=True,
```

Comment:
I think we actually just want to leave this out completely and let the `package_data` argument handle this. This argument has caused a lot of confusion in general because it only works with files that are under version control and/or in a `MANIFEST.in` file. That covers our case here, but I usually prefer to just use `package_data` so that we're explicit about what we're including.

Reply:
You're right. It's working for me with this argument removed.